src/components/shared/page.tsx

```diff
@@ -136,6 +138,18 @@ function ArticleBody({ capi, imageSalt, getAssetLocation }: BodyProps): ElementW
         ), resources: [articleScript] };
     }
 
+    if (item.design === Design.Media) {
+        const body = partition(item.body).oks as Image[];
+        const content = insertAdPlaceholders(renderMedia(imageSalt)(item.pillar, body));
```

Comment:
I think we might need a new `insertAdPlaceholders` here. We can remove it for the time being and create another ticket possibly to add adverts to galleries?